nucypher/characters/control/specifications/fields/messagekit.py

```diff
@@ -0,0 +1,15 @@
+from marshmallow import fields
+from base64 import b64decode, b64encode
+
+
+class MessageKitField(fields.Field):
```

Comment:
I guess this is really `UmbralMessageKit` unless you intend to use the same field type for `RevocationKit`...?